ArduCopter/mode_stabilize_heli.cpp

```diff
@@ -72,7 +72,13 @@ void ModeStabilize_Heli::run()
     }
 
     // call attitude controller
-    attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(target_roll, target_pitch, target_yaw_rate);
+    if (auto_yaw.mode() != AUTO_YAW_FIXED) {
```

Comment:
This file is actually for the traditional helicopter configuration.  I don't think this change is necessary (unless we're going to be putting the gimbal on helis, but that would likely require other changes to the codebase as well)

Reply:
You're right, this isn't needed.  I've removed the changes.